frontend/libs/studio-components/src/components/StudioInputTable/Cell/Cell.module.css

```diff
@@ -12,8 +13,24 @@
   display: inline-flex;
 }
 
-.textfieldCell:not(:hover) input:not(:hover):not(:active):not(:focus),
-.textareaCell:not(:hover) textarea:not(:hover):not(:active):not(:focus) {
+:is(
+    .textfieldCell:not(:hover) input,
+    .textareaCell:not(:hover) textarea,
+    .textResourceCell:not(:hover) input
+  ):not(:hover):not(:active):not(:focus),
+.textResourceCell:not(:hover) div:has(input:not(:hover):not(:active):not(:focus)) {
   background-color: transparent;
   border-color: transparent;
 }
+
+.textResourceCell:not(:hover) .textInput div:has(input:not(:hover):not(:active):not(:focus)) svg {
+  visibility: hidden;
+}
+
+.textResourceCell:not(:hover):not(:focus-within) .toggle {
+  visibility: hidden;
+}
+
+.textResourceCell .currentTextId {
+  padding-inline: var(--fds-spacing-2);
+}
```

Comment:
I agree. Thanks!

Reply:
I've fixed this now, but with `padding-inline-start` instead of `padding-left`. For our purposes it probably doesn't matter (we don't have any plans on translating to a right-to-left language), but it's more semantically correct.

Reply:
I've made [some additional changes](https://github.com/Altinn/altinn-studio/pull/14217/commits/49000d1e2796fda58b9a21870aada05b7909897a) in order to increase the size between the text fields when there are multiple beside each other (which will be the case for code lists).

**Before:**
![Skjermbilde 2024-12-18 113420](https://github.com/user-attachments/assets/538d5752-295a-438f-9e42-b7e7b4ce1481)

**After:**
![Skjermbilde 2024-12-18 113926](https://github.com/user-attachments/assets/613dbe9a-1340-41d2-83bc-d894bfa36f89)

What do you think?

Reply:
`padding-inline` is a quite new concept in CSS, so that's why it's so sparsely used. But it has been available in all major browsers for a couple of years now, so I think we should start using it more consistently.